lucille-core/src/main/java/com/kmwllc/lucille/stage/AddRandomField.java

```diff
@@ -0,0 +1,166 @@
+package com.kmwllc.lucille.stage;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.kmwllc.lucille.core.ConfigUtils;
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+import com.kmwllc.lucille.util.FileUtils;
+import com.typesafe.config.Config;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+
+/**
+ * Adds random data to a document field given parameters
+ * <br>
+ * Config Parameters -
+ * <br>
+ * <p>
+ * <b>inputDataPath</b> (String, Optional) : file path to a text file that stores datapoints to be randomly placed into field,
+ *  defaults to numeric data based on cardinality
+ * </p>
+ * <p>
+ * <b>fieldName</b> (String, Optional) : Field name of field where data is placed, defaults to "data"
+ *  </p>
+ * <p>
+ * <b>cardinality</b> (int, Optional) : size of the subset of datapoints to be grabbed either from
+ *  given datapath or from random numbers
+ * </p>
+ *  <p>
+ * <b>minNumOfTerms</b> (Integer, Optional) : minimum number of terms to be in the field, defaults to null
```

Comment:
comment should explain the behavior that a caller of this method would observe
in that sense, even though there's an initial default of null, the actual default is 1, right?

Reply:
changed to be 1 instead of null